src/Components/PetContainer.js

```diff
@@ -4,19 +4,27 @@ import '../Styles/PetContainer.scss';
 
 
 const PetContainer = ({ pets }) => {
-  const petCards = pets.map(pet => {
-    return (
-      <PetCard
-        id={pet.id}
-        name={pet.name}
-        image={pet.image}
-      />
-    )
-  });
+  let petCards;
+  let emptyMessage;
+
+  if(!pets.length){
```

Comment:
Nice work on this accounting for the sad paths if there are no pets for adoption.